technic/machines/HV/quarry.lua

```diff
@@ -201,11 +201,15 @@ local function get_dig_pos(quarry_pos, quarry_dir, dig_pos, dig_index, dig_steps
 		max_depth = quarry_max_depth
 	end
 
+	local dig_level = meta:get_int("dig_level")
+	if (quarry_pos.y - dig_level) > max_depth then
```

Comment:
Is this correct or should it be like how it was: `if (quarry_pos.y - dig_level) >= max_depth then`?
I mean more than vs more than or equal, without reading through all of it my first impression would be that new change is correct and old check was off by one but not sure... @OgelGames you know better probably?

Reply:
I tested this, both if clauses are usefull for those both cases:
The first if check with > makes sure the quarry goes not to deep in case it finds only vacuum and air.

The second if makes sure, that if there is something found insinde max_depth the quarry does not dig one block in the center from max_depth+1 layer bellow.

Reply:
It's correct as it is, because the quarry still needs to dig nodes at max depth.